src/lake/Lake/CLI/Init.lean

```diff
@@ -93,13 +93,13 @@ def mathConfigFileContents (pkgName libRoot : String) :=
 s!"import Lake
 open Lake DSL
 
-def moreServerArgs := #[
-  \"-Dpp.unicode.fun=true\", -- pretty-prints `fun a ↦ b`
-  \"-Dpp.proofs.withType=false\"
+def moreServerOptions : Array ServerOption := #[
+  ⟨`pp.unicode.fun, true⟩, -- pretty-prints `fun a ↦ b`
+  ⟨`pp.proofs.withType, false⟩
 ]
 
 -- These settings only apply during `lake build`, but not in VSCode editor.
-def moreLeanArgs := moreServerArgs
+def moreLeanArgs := moreServerOptions.map ServerOption.asCliArg
```

Comment:
Do we actually need this now? Lean should be passed these options automatically. This should be the default behavior, any option that affects the build has to be added in both places.

Reply:
I'm not saying there should not be a split, but people shouldn't have to repeat themselves and a place to put the common data would be useful for the common case.

Reply:
If both of you are happy with `leanOptions`, then I will add it.